src/EventListener.swift

```diff
@@ -10,14 +10,14 @@ public class EventListener <T> : Listener {
     if event._listeners[_targetID] == nil {
       event._listeners[_targetID] = [ptr]
     } else {
-      event._listeners[_targetID]!.append(ptr)
+      event._listeners[_targetID]?.append(ptr)
```

Comment:
Note that this class is **not** thread-safe, so if this line has crashed for you, then it's your own race condition.